ext/GeometryOpsProjExt/reproject.jl

```diff
@@ -25,9 +25,16 @@ end
 function reproject(geom, source_crs, target_crs;
     time=Inf,
     always_xy=true,
-    transform=Proj.Transformation(convert(String, source_crs), convert(String, target_crs); always_xy),
+    transform=nothing,
     kw...
 )
+    transform = if isnothing(transform) 
+        s = source_crs isa Proj.CRS ? source_crs : convert(String, source_crs)
```

Comment:
Should this be something like `to_proj_consumable(source_crs)` instead, which can then have multiple dispatches?

Reply:
We can also add that later, since I want to figure out a good way to incorporate other transformation packages into this as well.

Reply:
Yeah I thought about that too, but since its in a Proj extension right no thats not so easy